app/lib/methods/handleBase64.ts

```diff
@@ -0,0 +1,21 @@
+export const imageBase64RegExp = new RegExp(/^data:image\/[bmp,gif,ico,jpg,png,svg,webp,x\-icon,svg+xml]+;base64,/);
+const regExpOf120Characters = new RegExp(/^data:image\/[bmp,gif,ico,jpg,png,svg,webp,x\-icon,svg+xml]+;base64,([\w+\/=]{1,120})/);
+const mimeTypeBase64RegExp = new RegExp(/^data:(.+);base64,/);
+
+export function isImageBase64(data: string): boolean {
+	return !!data && imageBase64RegExp.test(data);
+}
+
+export function valueOfFirst120CharactersOfImageBase64(imageBase64: string): string | undefined {
+	const result = imageBase64.match(regExpOf120Characters)?.[0];
+	if (result) {
+		return result.replace(imageBase64RegExp, '');
+	}
+}
+
+export function getBase64MimeType(data: string): string | undefined {
+	const mimeType = data.match(mimeTypeBase64RegExp);
+	if (mimeType) {
+		return mimeType[1];
+	}
+}
```

Comment:
Where did you get this from?

Reply:
The regExp I found here: https://regex101.com/library/XAQEkg?filterFlavors=javascript&filterFlavors=golang&orderBy=MOST_RECENT&search=

I just modified a bit to get the 120 first characters to avoid testing all the data as react-native-ui-lib does here: https://github.com/wix/react-native-ui-lib/blob/32ad53c9d864c34d0df35c853b5c9f82c234dff3/src/utils/imageUtils.ts#L14-L18

Add to this, I selected 120 characters after the header of base64 to differentiate between the images, but this value is in my opinion.